addon/kubeflow-operator/manifest.yaml

```diff
@@ -164,19 +164,11 @@ spec:
   - kustomizeConfig:
       parameters:
       - name: clusterRbacConfig
-        value: "OFF"
+        value: "ON"
```

Comment:
What is this `clusterRbacConfig` for? Do you have an idea :) ? I didn't find anything about this in kubeflow docs

Reply:
https://istio.io/latest/zh/docs/reference/config/security/istio.rbac.v1alpha1/#RbacConfig-Mode

Reply:
Yes, I know, but why do we need this for dex?

Reply:
I think we need it for multi-tenancy

Reply:
Right, it's not that we would need it, we want it for true multi-tenancy. Of course when authentication is disabled, it does not make sense to have it on.